README.md

```diff
@@ -1,3 +1,30 @@
-##### openshift-controller-manager
+# OpenShift Controller Manager
 
-TODO: Add content
+The openshift controller manager (a.k.a OCM) reconciles the status of all OpenShift API objects. The
```

Comment:
The OpenShift Controller Manager (OCM) is comprised of multiple controllers, each of which corresponds to a top-level OpenShift API object, watching for changes and acting accordingly.